packages/venia-ui/lib/components/AccountMenu/accountMenuItems.js

```diff
@@ -25,6 +16,25 @@ const AccountMenuItems = props => {
 
     const classes = mergeClasses(defaultClasses, props.classes);
 
+    const intl = useIntl();
+    const MENU_ITEMS = [
+        {
+            name: intl.formatMessage({ id: 'Order History' }),
+            url: '/order-history'
+        },
+        {
+            name: intl.formatMessage({ id: 'Store Credit & Gift Cards' }),
+            url: ''
+        },
+        { name: intl.formatMessage({ id: 'Favorites Lists' }), url: '' },
+        { name: intl.formatMessage({ id: 'Address Book' }), url: '' },
+        { name: intl.formatMessage({ id: 'Saved Payments' }), url: '' },
+        {
+            name: intl.formatMessage({ id: 'Communications' }),
+            url: '/communications'
+        },
+        { name: intl.formatMessage({ id: 'Account Information' }), url: '' }
+    ];
```

Comment:
Yes, we do need to translate these, but we should probably continue to define `MENU_ITEMS` statically (before the component) and just call `formatMessage` from the same map operation we're already doing to create elements.

We might also take this opportunity to give these static items keys that more uniquely identify them within the app (e.g., `accountMenu.orderHistoryLink`), since that kinda seems to be what `react-intl` expects us to do. Thoughts?

```es6
const MENU_ITEMS = [/* ... */]

const AccountMenuItems = props => {
  // ...

  const { formatMessage } = useIntl()

  // could call `intl.formatMessage(item.name)` on each
  // but we might as well just use `FormattedMessage`, no?
  const menuItems = MENU_ITEMS.map(item => (
    <Link key={item.name} className={classes.link} to={item.url}>
      <FormattedMessage id={item.name} defaultMessage={item.name} />
    </Link>
  )
}
```